Lists/Lab/ListsOfProducts/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ListsOfProducts
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int number = int.Parse(Console.ReadLine());
+            List<string> finalArr = new List<string>();
+
+            for (int i = 0; i < number; i++)
+            {
+                finalArr.Add(Console.ReadLine());
+            }
+
+            SortedProducts(finalArr);
+        }
+
+        static void SortedProducts(List<string> finalArr)
```

Comment:
SortProducts if it's going to be a void method. I'd suggest returning a sorted list and modifying the parameter